src/app/libp2p_helper/src/libp2p_helper/main.go

```diff
@@ -46,6 +46,13 @@ func min(a, b uint64) uint64 {
 const MAX_MESSAGE_LENGTH uint64 = 2 << 30  // 2gb
 const MESSAGE_BUFFER_SIZE uint64 = 2 << 20 // 2mb
 
+type streamState int
+
+const (
+	STREAM_QUERY_UNHANDLED streamState = iota
```

Comment:
The tense here threw me a little; might be clearer with e.g. `STREAM_DATA_EXPECTED`/`STREAM_DATA_UNEXPECTED`?